### DISCUSSION TITLE
Posts of the status type are not excluded from RSS feed

### DISCUSSION THREAD

**Discussion by igor-baiborodine** [Lednerb/bilberry-hugo-theme#337] [Q&A] [answered]
@Lednerb, currently posts of the `status` type are excluded from indexing for Algolia:
![image](https://user-images.githubusercontent.com/1027701/150606248-d34e4f07-1eeb-4f56-b90a-5d714b118513.png)

Therefore, it would be logical to exclude them from the RSS feed, which is not currently the case:
![image](https://user-images.githubusercontent.com/1027701/150606370-c03ea599-cea1-4b2e-beed-18cf5d980225.png)

What do you think?

**Comment by Lednerb** [comment, marked as answer]
Hi @igor-baiborodine,


In my opinion the `status` type can be used for very short postings / moods / announcements whereas the search functionality will not benefit from searching the `status` posts.

RSS feeds in opposite are used differently and for the purpose of _staying in contact_ or getting information about the newest announcements etc.

Therefore I would prefer keeping the `status` posts in the RSS feed as it's currently implemented.
